spec/mailers/order_assignment_mailer_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.describe OrderAssignmentMailer do
+  let(:assigned_user) { FactoryGirl.build(:user) }
+  let(:email) { ActionMailer::Base.deliveries.last }
+  let(:facility) { FactoryGirl.build_stubbed(:facility) }
+
+  def stubbed_order_detail(order)
+    FactoryGirl.build_stubbed(:order_detail,
+                              assigned_user: assigned_user,
+                              order: order)
+  end
+
+  describe ".notify_assigned_user" do
+    context "when given a single order_detail" do
+      let(:order) { FactoryGirl.build_stubbed(:order, facility: facility) }
+      let(:order_detail) { stubbed_order_detail(order) }
+
+      before { described_class.notify_assigned_user(order_detail).deliver_now }
+
+      it "generates an order assignment notification", :aggregate_failures do
+        expect(email.to).to eq [assigned_user.email]
+        expect(email.subject).to include("Order Assignment Notice")
+        expect(email.html_part.to_s)
+          .to include(order_detail.to_s)
+          .and include("assigned this&nbsp;order")
```

Comment:
What's the value of doing this check? It's especially weird to me because it's testing for the presence of the &nbsp; which is generated by text_helpers.